dl/Util.py

```diff
@@ -316,6 +316,40 @@ def split_auth_token(token):
     return res.groups() if res else None
 
 
+def auth_token2dict(token):
```

Comment:
Consider the name `auth_token_to_dict` or `token_to_dict` (primarily just the `to_dict` portion). My thought is that will make it more consistent with methods in other packages (some classes offer `to_dict` methods.)